src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -0,0 +1,52 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.response.OrderResponseDto;
+import mate.academy.spring.mapper.impl.response.OrderResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.User;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final UserService userService;
+    private final ShoppingCartService shoppingCartService;
+    private final OrderResponseMapper orderResponseMapper;
+
+    public OrderController(OrderService orderService,
+                           UserService userService,
+                           ShoppingCartService shoppingCartService,
+                           OrderResponseMapper orderResponseMapper) {
+        this.orderService = orderService;
+        this.userService = userService;
+        this.shoppingCartService = shoppingCartService;
+        this.orderResponseMapper = orderResponseMapper;
+    }
+
+    @GetMapping
+    public List<OrderResponseDto> getAllOrderByUserId(@RequestParam Long userId) {
```

Comment:
```suggestion
    public List<OrderResponseDto> getOrdersHistor(@RequestParam Long userId) {
```
keep to consistency with service layer method